### ISSUE TITLE
the problem of rsDriver

### ISSUE THREAD

**Issue by chhluo** [ropensci/wdman#12]
Please fill the following template:

### Operating System
Win10
R x64 3.3.3

### Please describe Expected behaviour
I ran the selenium-server-standalone-3.3.1.jar by windows cmd,and it is running.
then I want to run firefox by using following command :
   library(RSelenium)
   rsDriver(browser = "firefox")

### Please describe Actual behaviour

checking Selenium Server versions:
BEGIN: PREDOWNLOAD
Error in open.connection(con, "rb") : Couldn't connect to server
![Uploading 1.png…]()
![Uploading 2.png…]()

**Comment by johndharrison** [comment]
If you are running a standalone server you don't need to run rsDriver. I assume you ran the standalone server on port 4444. In which case the following should work:

```
remDr <- remoteDriver()
remDr$open()
remDr$navigate("http://www.r-project.org")
...
...
remDr$close()
```

**Comment by chhluo** [comment]
I ran the the standalone server on the port 4444,it worked.But a another problem came.Pelease see the following picture.
And i tried to install  geckodriver package,but it is not available for R version 3.3.3.Is there any way to solve the problem?
![1](https://cloud.githubusercontent.com/assets/26483343/24330353/9a2a2482-124e-11e7-8a82-30532ccf56b2.png)
![2](https://cloud.githubusercontent.com/assets/26483343/24330352/99f98a5c-124e-11e7-9823-53ad865d4b65.png)
